styles/common.module.scss

```diff
@@ -0,0 +1,111 @@
+@timesnewroman: "Times New Roman", Times, Serif;
+@helvetica: "Helvetica Neue", Helvetica, Arial;
+@georgia: Georgia, Times, "Times New Roman";
+
+.container {
+    min-height: 100vh;
+    padding: 0 0.5rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.main {
+    max-width: 1440px;
+    width: 100%;
+    padding: 1.5rem 0;
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    font-family: helvetica;
+
+    h3 {
+        font-family: georgia;
+        font-weight: bold;
+    }
+
+    hr {
+        width: 80%;
+        height: 1px;
+        background-color: #c8c8c8;
+    }
+
+    ul {
+        text-align: left;
+    }
+
+    a {
+        text-decoration: underline;
+    }
+}
+
+.title {
+    margin: 0;
+    line-height: 1.15;
+    font-size: 3rem;
+    margin-top: 1.5rem;
+    font-family: timesnewroman;
+
+    a {
+        color: #0070f3;
+        text-decoration: none;
+
+        &:hover,
+        &:focus,
+        &:active {
+            text-decoration: underline;
+        }
+    }
+}
+
+.title,
+.description {
+    text-align: center;
+}
+
+.card {
+    width: 80%;
+    margin: 1rem;
+    flex-basis: 45%;
+    padding: 1.5rem;
+    text-align: center;
+    color: inherit;
+    text-decoration: none;
+    transition: color 0.15s ease, border-color 0.15s ease;
+
+    h3 {
+        margin: 0 0 1rem 0;
+        font-size: 1.75rem;
+    }
+
+    p {
+        margin: 0;
+        font-size: 1.125rem;
+        line-height: 1.5;
+        text-align: left;
+    }
+}
+
+.imgwithtext {
```

Comment:
Please remove any styles in this file that are not used or referenced in the code.

Reply:
done